api/management/commands/sync_molnix.py

```diff
@@ -41,6 +46,23 @@
 }
 
 
+def get_molnix_status(position_status):
+    """
+    A position_status of active should be shown as Open
+    A position_status of archived should be shown as Closed
+    A position_status of unfilled should be shown as Stood Down
+    If the position_status is non other than active, archived, unfilled then show Closed.
+    """
+    molnix_status_dict = {
+        "active": SurgeAlertStatus.OPEN,
+        "unfilled": SurgeAlertStatus.STOOD_DOWN,
+        "archived": SurgeAlertStatus.CLOSED,
+    }
+    if molnix_status_dict.get(position_status, None) is not None:
+        return molnix_status_dict.get(position_status)
+    return SurgeAlertStatus.CLOSED
```

Comment:
Let's move this to SurgeAlert as staticmethod and let's change it to this

```suggestion
def parse_molnix_status(status_raw: str) -> SurgeAlertStatus:
    """
    A position_status of active should be shown as Open
    A position_status of archived should be shown as Closed
    A position_status of unfilled should be shown as Stood Down
    If the position_status is non other than active, archived, unfilled then show Closed.
    """
    molnix_status_dict = {
        "active": SurgeAlertStatus.OPEN,
        "unfilled": SurgeAlertStatus.STOOD_DOWN,
        "archived": SurgeAlertStatus.CLOSED,
    }
    return molnix_status_dict.get(position_status.lower(), SurgeAlertStatus.CLOSED)
```